tests/source/configs-fn_call_style-block.rs

```diff
@@ -1,6 +1,9 @@
 // rustfmt-fn_call_style: Block
+// rustfmt-max_width: 120
```

Comment:
I would prefer not to change max-width in this test, instead we ought to be able to change the code to work at the default width